components/Feature/StatusForm/index.module.css

```diff
@@ -0,0 +1,3 @@
+#referral-rejection-reason {
+  margin-bottom: 0;
+}
```

Comment:
Could we be able to use govuk margin class for this?

Reply:
Maybe, will try. Haven't thought of that for some reason.